cas-server-support-x509/pom.xml

```diff
@@ -54,10 +54,10 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <artifactId>vt-crypt</artifactId>
-      <groupId>edu.vt.middleware</groupId>
-      <version>2.1.4</version>
-    </dependency>
+	  <groupId>org.cryptacular</groupId>
```

Comment:
To be clear, the suggestion is to move:

```
<dependency>
	  <groupId>org.cryptacular</groupId>
	  <artifactId>cryptacular</artifactId>
	  <version>${cryptacular.version}</version>
</dependency>
```

To the parent POM, and then declare the following in the x509 module:
```
<dependency>
	  <groupId>org.cryptacular</groupId>
	  <artifactId>cryptacular</artifactId>
</dependency>
```